trigger/testTimer/README.md

```diff
@@ -0,0 +1,80 @@
+<!--
+title: CSV Timer
+
+-->
+# Timer Trigger
+This trigger provides your flogo application the ability to get data from a CSV file after a particular interval for an action 
+
+## Installation
+
+```bash
+flogo install github.com/skothari-tibco/csvtrigger
+```
+
+## Metadata
+```json
+{
+  "handler": {
+    "settings": [
+      {
+        "name": "filePath",
+        "type": "string"
+      },
+      {
+        "name": "repeatInterval",
+        "type": "string"
+      }
+    ]
+  }
+}
+```
+### Details
+
+#### Settings:
+| Setting  | Required | Description |
+|:---------|:---------|:------------|
+| control  | false    | Enable control of trigger
+| port     | false    | Port number on which control api is called (/control/resume, /control/pause, /control/restart)
+
+#### Handler Settings:
+| Setting        | Required | Description |
+|:---------------|:---------|:------------|
+| filePath       | true     | Path to a CSV file
+| repeatInterval | false    | the repeat interval (1, 200 etc in millisecond), doesn't repeat if not specified
+| id             | true     | Id of Handler
```

Comment:
There is already a handler id as part of the handler object.  You don't need this.